File: Client/Game_CrazyShake/src/Dto/GameCommand/ClientInitDto.ts
namespace Dto {
    /**
     * 初始化游戏Dto
     */
    export class ClientInitDto {
        /**
         * 赔率信息
         */
       public PosOdds:Object;
       /**
        * 用户余额
        */
       public Balance:number;
       /**
        * 最大投注限额
        */
       public MaxBet:number;
    }
}

File: QIC.EGame.GameMachineService/QIC.EGame.GameMachineService.CrazyShake/Dtos/ClientInitDto.cs
using System.Collections.Generic;
using QIC.EGame.GameMachineService.CrazyShake.Enums;

namespace QIC.EGame.GameMachineService.CrazyShake.Dtos
{
    /// <summary>
    /// 客户端初始化的Dto
    /// </summary>
    public class ClientInitDto
    {
        /// <summary>
        /// 赔率信息
        /// </summary>
        public Dictionary<BetPosType, decimal> PosOdds { get; set; }

        /// <summary>
        /// 用户余额
        /// </summary>
        public decimal Balance { get; set; }
    }
}
